src/main/java/com/myaudiolibrary/web/model/Album.java

```diff
@@ -0,0 +1,77 @@
+package com.myaudiolibrary.web.model;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class Album {
+
+    @ManyToOne
+    @JoinColumn(name = "ArtistId")
+    private Artist artist;
+
+    @Id
+    @Column(name = "AlbumId")
+    private Long id;
+
+    @Column(name = "Title")
+    private String title;
+
+    public Album() {
+
+    }
+
+    public Album(Artist artist, String title, Long id) {
+        this.artist = artist;
+        this.title = title;
+        this.id = id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public Artist getArtist() {
+        return artist;
+    }
+
+    public void setArtist(Artist artist) {
+        this.artist = artist;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Album album = (Album) o;
+        return Objects.equals(artist, album.artist) &&
```

Comment:
Il faut que tu enlèves `artist` de `equals()` et `hashCode()`, c'est cela qui provoque la récursion...
```suggestion
        return 
```